.circleci/config.yml

```diff
@@ -68,20 +68,67 @@ commands:
 # Define a job to be invoked later in a workflow.
 # See: https://circleci.com/docs/2.0/configuration-reference/#jobs
 jobs:
-  build-and-test:
+  init:
     docker:
       - image: sematicai/sematic-ci:latest
     steps:
       - checkout
       - install-ci-pip-deps
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - .pyenv
+            - project
+
+  build-and-lint:
```

Comment:
Is the result of the build persisted in the workspace via lateral effect? I'm not seeing any explicit `persist_to_workspace` here.
Does that actually mean that the unit tests also re-build in parallel?

<img width="450" alt="image" src="https://user-images.githubusercontent.com/1894533/216190865-5a49132a-476c-4096-b14d-ef615e2d8446.png">

Should we instead have a pure `build` job that is requirement for `unit-tests`, `integration-test`, and a separated-out `lint` job as well?

Reply:
Yes, `unit tests also re-build in parallel` is true.   The name `build-and-lint` is a mistake here. It doesn't build. I will fix.

Reply:
Fixed now. Screenshot is updated in the description. PTAL.